PhysicsTools/NanoAOD/python/nano_cff.py

```diff
@@ -154,11 +154,28 @@ def nanoAOD_activateVID(process):
 def nanoAOD_customizeCommon(process):
 
     process = nanoAOD_activateVID(process)
+    
+    # Include new ParticleNet trainings for Run-2 UL
+    run2_nanoAOD_106Xv2.toModify(
+        nanoAOD_addDeepInfoAK4CHS_switch, nanoAOD_addParticleNet_switch=True, 
+    )
```

Comment:
Note that you can chain modifier commands i.e. `run2_nanoAOD_106Xv2.toModify(...).toModify(...)`